.storybook/preview.js

```diff
@@ -81,16 +83,19 @@ const themeDecorator = (story, { globals, parameters }) => {
   switch (currentTheme) {
     case THEMES.NG_LIGHT:
       styleElement.textContent = clrUiStyles;
+      document.body.removeAttribute('cds-text');
```

Comment:
I think we should add a `cdsTextAttribute` constant like `cdsThemeAttribute` instead of repeating this string.